apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/StargateRequestInfo.java

```diff
@@ -42,18 +41,16 @@ public class StargateRequestInfo {
 
   private final Optional<String> cassandraToken;
 
-  private final StargateBridge stargateBridge;
+  private final StargateBridge stargateBridge = null;
 
   @Inject
   public StargateRequestInfo(
       RoutingContext routingContext,
       SecurityContext securityContext,
-      GrpcClients grpcClients,
```

Comment:
for the sake of the backward compatibility, you should inject the `@GrpcClient("bridge") StargateBridge bridge` and set the field, otherwise bunch of code will report NPE atm..